src/Parser/Warlock/Affliction/Modules/Features/Abilities.js

```diff
@@ -8,135 +8,119 @@ class Abilities extends CoreAbilities {
     const combatant = this.combatants.selected;
     return [
       // Rotational spells
-      {
-        spell: SPELLS.HAUNT_TALENT,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 25,
-        enabled: combatant.hasTalent(SPELLS.HAUNT_TALENT.id),
-        isOnGCD: true,
-        castEfficiency: {
-          suggestion: true,
-          recommendedEfficiency: 0.95,
-          extraSuggestion: 'This estimate may not be correct sometimes because of Haunt\'s resets. The real amount of possible Haunts will be higher if there were adds on this fight.',
-        },
-      },
-      {
-        spell: SPELLS.PHANTOM_SINGULARITY_TALENT,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: 40,
-        enabled: combatant.hasTalent(SPELLS.PHANTOM_SINGULARITY_TALENT.id),
-        isOnGCD: true,
-        castEfficiency: {
-          suggestion: true,
-        },
-      },
       {
         spell: SPELLS.AGONY,
-        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        category: Abilities.SPELL_CATEGORIES.DOTS,
```

Comment:
This category does not exist at the moment: https://github.com/WoWAnalyzer/WoWAnalyzer/blob/ecbb94d9e63f7a2263872522addf975417be9243/src/Parser/Core/Modules/Abilities.js#L14
Any reason to make the distinction between rotational and dot? dots seems pretty rotational to me :P